build.proj

```diff
@@ -162,6 +162,13 @@
     <CallTarget Targets="CodeSignBinaries" Condition=" '$(CodeSign)' == 'true' " />
     
     <Message Importance="high" Text="Building Setup..." />
+    <CallTarget Targets="BuildInstaller" 
+                Condition=" '$(CodeSign)' == 'true' and '$(Scope)' == 'all'" />
+  </Target>
+  
+    <!-- Build all flavors of the installers -->
+  <Target Name="BuildInstaller" DependsOnTargets="RestoreNugetPackages;BuildMsBuildTask">
+    <Message Importance="high" Text="Building Installers..." />    
     <MSBuild
```

Comment:
How are the modules/dlls get into the "Package" folder if we don't call either BuildInstaller.ps1 or SaveModules.ps1 here?